src/components/CaptureDetailDialog.js

```diff
@@ -115,23 +135,35 @@ function CaptureDetailDialog(props) {
     }
   }, [cdContext.capture, capture]);
 
+  function handleCaptureTagDeletion(tagId) {
+    console.log(`TODO: delete tag w/ id: ${tagId}`);
+    setCaptureTagDeletionTarget(undefined);
+  }
+
   function handleClose() {
     setSnackbarOpen(false);
     setSnackbarLabel('');
+    setCaptureTagDeletionTarget(undefined);
     cdContext.reset();
     props.onClose();
   }
 
   function Tags(props) {
     const { capture, species, captureTags } = props;
-    const allTags = [
+    const otherTags = [
       capture.morphology,
       capture.age,
       capture.captureApprovalTag,
       capture.rejectionReason,
-      ...captureTags.map((t) => t.tagName),
```

Comment:
This should contain deletable tags, but only gets populated after the dialog is opened as the tag names are queried from the server.

Reply:
Understood, I believe that the `otherTags` collection is currently only string values that get rendered. I'm separating the capture tags to render with respective api IDs in preparation for the deletion call on removal confirmation.